mypy/checker.py

```diff
@@ -1063,6 +1064,20 @@ def enter_attribute_inference_context(self) -> Iterator[None]:
         yield None
         self.inferred_attribute_types = old_types
 
+    def _is_empty_generator(self, func: FuncItem) -> bool:
```

Comment:
```suggestion
    @staticmethod
    def _is_empty_generator(func: FuncItem) -> bool:
```

Or even better: move the function into the global scope?

Reply:
e31efd3a07d08af89715190484e77bdc1d7f2614